### ISSUE TITLE
Validation Data Missing Epitope Label

### ISSUE THREAD

**Issue by sidhomj** [phbradley/tcr-dist#29]
The current file with the validation cohort does not have the epitope label. It instead has EPI for all clones. 

![image](https://user-images.githubusercontent.com/27970764/53392195-accbde00-3966-11e9-8005-ddc6d43a6cdf.png)

**Comment by phbradley** [comment]
Hi there-- thanks for pointing this out. As I recall, for this dataset not all of the TCRs are specific for one of the four epitopes; these cells were stained with four tetramers and index sorted, so we have MFI values for all four (the *_MFI fields toward the end). Given a threshold on those MFI values, one can define a set of positive binders. For the majority of the original dataset we did not have this multi-tetramer MFI information, which explains why this file is structured a little differently. Let me know if that's not clear. Take care, Phil.

**Comment by sidhomj** [comment]
Thanks! I figured it out eventually. Should have closed the issue. Also, does tcr-dist take amino acid sequences directly? Or only nuc?

JW

Sent from my iPhone

On Feb 26, 2019, at 10:01 AM, Philip Bradley <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi there-- thanks for pointing this out. As I recall, for this dataset not all of the TCRs are specific for one of the four epitopes; these cells were stained with four tetramers and index sorted, so we have MFI values for all four (the *_MFI fields toward the end). Given a threshold on those MFI values, one can define a set of positive binders. For the majority of the original dataset we did not have this multi-tetramer MFI information, which explains why this file is structured a little differently. Let me know if that's not clear. Take care, Phil.
